### ISSUE TITLE
Lua Script documentation

### ISSUE THREAD

**Issue by robertoalmeida** [FrSkyRC/ETHOS-Feedback-Community#2142]
Where can I find documentation on the methods for creating a lua script?

**Comment by mawzthefinn** [comment]
You can find ETHOS-specific Lua information in the Lua-doc file here:

https://github.com/FrSkyRC/ETHOS-Feedback-Community/releases/tag/1.4.0

There's a good guide to getting tarted with Lua for OpenTX here:

http://rc-soar.blogspot.com/2014/07/getting-up-to-speed-with-lua.html

Almost all of it also applies to ETHOS

And finally there's the RCGroups thread for Lua on ETHOS

https://www.rcgroups.com/forums/showthread.php?4018791-FrSky-ETHOS-Lua-Script-Programming

**Comment by robertoalmeida** [comment]
Is it possible to get the flightmode name?

**Comment by mawzthefinn** [comment]
I don't believe so right now, there's an open issue, #1828 for that functionality

**Comment by mawzthefinn** [comment]
I've just been informed that there is Lua documentation available in the Documentation tab in ETHOS Suite 1.4.3 or later.

There is an update on #1828 for a call which should return flight mode name
